public/docs/v0/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <title>Documentation for v0</title>
+    <link href="v2/style.css" media="screen" rel="stylesheet" type="text/css">
+  </head>
+  <body>
+    <div id="container">
+      <div id="operations">
+        <h3>API Operations</h3>
+      </div>
+      <div id="content">
+        <h1>Documentation for v0</h1>
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
I'm not sure if we need to ship this with every install. Some wiki pages or creating api.diasporafoundation.org should be good enough.